app/main.development.js

```diff
@@ -394,11 +395,8 @@ const launchDCRWallet = () => {
   }
 
   // Add any extra args if defined.
-  if (argv.extrawalletargs != undefined) {
-    var extraArgs = argv.extrawalletargs.split(" ");
-    for (var i = 0; i < extraArgs.length; i++) {
-      args.push(extraArgs[i]);
-    }
+  if (argv.extrawalletargs !== undefined) {
+    args.concat(stringArgv(argv.extrawalletargs));
```

Comment:
`args.concat` just isn't working for me.  i tested with `args.push` and that seems to work as expected with multiple spaced args.  Maybe it's a OS/env issue?  linux amd64 debian8 VM

Reply:
hum, I'm on mac but should not be an issue because the concat method is a part of [es2015](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Array/concat) I will try to add a second verification `isString` and to use the `concat` from lodash, is it working for you?